android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/FingerPrintViewFactory.java

```diff
@@ -0,0 +1,185 @@
+package com.vijay.jsonwizard.widgets;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.os.Build;
+import android.text.TextUtils;
+import android.util.TypedValue;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import com.rey.material.util.ViewUtil;
+import com.vijay.jsonwizard.R;
+import com.vijay.jsonwizard.constants.JsonFormConstants;
+import com.vijay.jsonwizard.fragments.JsonFormFragment;
+import com.vijay.jsonwizard.interfaces.CommonListener;
+import com.vijay.jsonwizard.interfaces.FormWidgetFactory;
+import com.vijay.jsonwizard.interfaces.JsonApi;
+import com.vijay.jsonwizard.utils.FormUtils;
+import com.vijay.jsonwizard.utils.ImageUtils;
+import com.vijay.jsonwizard.utils.ValidationStatus;
+import com.vijay.jsonwizard.views.JsonFormFragmentView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by vijay on 24-05-2015.
+ */
+public class FingerPrintViewFactory implements FormWidgetFactory {
+
+    public static int dp2px(Context context, float dp) {
+        Resources r = context.getResources();
```

Comment:
Please name variables accordingly for increased code readability

Reply:
ok will change actually was copied from another class ImagePickerViewFactory.

Reply:
No problem. We have gotten the time to update the code standards in all the classes but slowly we shall update them.